docs/plugins/ingest.asciidoc

```diff
@@ -403,57 +403,110 @@ An example that adds the parsed date to the `timestamp` field based on the `init
 }
 --------------------------------------------------
 
-==== Meta processor
+=== Accessing data in pipelines
 
-The `meta` processor allows to modify metadata properties of a document being processed.
+Processors in pipelines have read and write access to documents that pass through the pipeline.
+The fields in the source of a document and its metadata fields are accessible.
 
-The following example changes the index of a document to `alternative_index` instead of indexing it into an index
-that was specified in the index or bulk request:
+Accessing a field in the source is straightforward and one can refer to fields by
+their name. For example:
 
 [source,js]
 --------------------------------------------------
 {
-  "description" : "...",
-  "processors" : [
```

Comment:
it used to be part of the `meta` processor config. I thought I could shorten the examples by only showing a processor, since the previous section shows entire pipeline examples.